profile/README.md

```diff
@@ -2,6 +2,7 @@
 # Welkom! 👋
 
 ## 🙋‍♀️ Wat is Klantinteractie Servicesysteem (kiss)
+Het Klantinteractie Servicesysteem (KISS) is  een applicatie waarmee Klantcontactmedewerkers (KCM) optimaal worden ondersteund in hun werk: het informeren en helpen van burgers en ondernemers die contact opnemen met de Gemeente. Doel van KISS is om alle informatie die een KCM nodig heeft voor die taak, zoveel mogelijk binnen KISS zelf te ontstluiten. Het is ontwikkeld als een Common Ground component in opdracht van [Dimpact](https://www.dimpact.nl/klantinteractie-servicesysteem) en andere samenwerkende gemeenten.
```

Comment:
Hey Sytske, super dat er hier naar gekeken wordt. 

Ik zou in de titel (kiss) ook nog naar all caps zetten

verder nog: 
*ontsluiten , staat nu ontstluiten nog